src/navsatfixex.py

```diff
@@ -0,0 +1,57 @@
+'''uint8 COVARIANCE_TYPE_UNKNOWN=0
```

Comment:
This file's purpose is now clear. It was made just to create NavSatFix messages so that they could be sent to the drone in simulation. This does not need to exist any more. I recommend this is deleted.